exercises/robot-simulator/src/lib.rs

```diff
@@ -0,0 +1,44 @@
+// The code below is a stub. Just enough to satisfy the compiler.
+// In order to pass the tests you can add-to or change any of this code.
+
+#[derive(PartialEq, Debug)]
+pub enum Direction {
+    North,
+    East,
+    South,
+    West,
+}
+
+pub struct Robot;
+
+impl Robot {
+    #[allow(unused_variables)]
+    pub fn new(x: isize, y: isize, d: &Direction) -> Self {
```

Comment:
To be clear, that means `fn new(x: isize, y: isize, d: Direction) -> Self` rather than the current signature with the push towards the student calling `clone` to take ownership?